server/src/test/java/org/elasticsearch/rest/action/search/RestSearchActionTests.java

```diff
@@ -68,4 +118,148 @@ public void testIllegalSearchType() {
         Exception ex = expectThrows(IllegalArgumentException.class, () -> action.prepareRequest(request, verifyingClient));
         assertEquals("No search type for [some_search_type]", ex.getMessage());
     }
+
+    public void testCCSForceFailFlag() throws IOException {
```

Comment:
I would write this type of test for every API that supports the flag. What do you think?